lib-rest-client/pmbhandle/rest.go

```diff
@@ -63,7 +63,7 @@ func ContactPlugin(url, method, token string, odataID string, body interface{},
 	}
 
 	if resp.StatusCode >= 300 {
-		log.Printf("warning: got %v, while fetching %v with method %v", resp.Status, url, method)
+		log.Warn("got %v, while fetching %v with method %v" + resp.Status + url + method)
```

Comment:
log.Warn("got %v, while fetching %v with method %v" + resp.Status + url + method)
can be changed to
log.Warnf("got %v, while fetching %v with method %v", resp.Status, url, method)

Reply:
Will change in future